lib/Importer/ONNX.cpp

```diff
@@ -63,6 +63,17 @@ bool ONNXModelLoader::loadProtoFile(onnx::GraphProto &net,
   onnx::ModelProto MP;
   bool parseNet = MP.ParseFromCodedStream(&codedstr);
   net = MP.graph();
+  ir_version = MP.ir_version();
```

Comment:
Let's move new code to new function. We don't want `loadProtoFile()` to be too long.